src/components/DateRangeInput/DateRangeInput.tsx

```diff
@@ -283,13 +289,17 @@ const DateRangeInput: React.FC<
               >
                 <Flex align="center" justify="center" spacing={3}>
                   <TimePicker
+                    helperText="From Time"
```

Comment:
can we rename `helperText` to `label`? In essence, that's what it is, since it should also force  a "focus" on the related input/combobox field.

In addition, it's a prop that's consistent with the rest of the components

Reply:
Hey, thanks for the comment. There is already a label prop in place.
https://github.com/panther-labs/pounce/blob/3e0988b0445d0334071385f1f7e5ff5063583536/src/components/DateInput/TimePicker.tsx#L14-L20

Reply:
Actually scratch that, we can use `label` as is. 🤗